<?xml version="1.0"?>
<launch>
  <master auto="start"/>

  <!-- Turn The Deadman On So The Robot Can Move -->
<!--  <node name="deadman_on" pkg="landshark_2dnav" type="deadman_turn_on.py"/> -->

  <!-- Convert cmd_vel to base_velocity -->
<!--  <node name="cmdvel_to_basevelocity" pkg="landshark_2dnav" type="cmdvel_to_basevelocity.py"/> -->

  <!-- pub odom stub -->
  <node name="pub_odom_dummy" pkg="landshark_2dnav" type="pub_odom_dummy.py"/> 

  <!-- Convert /landshark/gps to /landshark/gps_odom UTM -->
  <include file="$(find landshark_gps_meters)/launch/landshark_gps_meters.launch" /> 

  <!-- Run the map server -->
  <!--  <node name="map_server" pkg="map_server" type="map_server" args="$(find landshark_2dnav)/map/willow.pgm 0.025"/> -->

  <include file="$(find landshark_2dnav)/launch/map_server.launch" /> 
    
  <include file="$(find landshark_2dnav)/launch/ekf_template.launch" /> 

  <include file="$(find landshark_2dnav)/launch/amcl_diff.launch" />
  
  <include file="$(find landshark_2dnav)/launch/fake_localization.launch" />

  <include file="$(find landshark_2dnav)/launch/move_base_only.launch" />

  <!-- advertize waypoint service -->
<!--  <include file="$(find landshark_2dnav)/launch/navigate.launch" />-->
  <node name="navigate" pkg="landshark_2dnav" type="navigate.py"/>

  <include file="$(find landshark_description)/launch/landshark_description.launch"/>





<!--  <arg name="is_spoofable" value="$(optenv IS_SPOOFABLE true)" />-->

  <!-- Spoofers -->
<!--  <group if="$(arg is_spoofable)">-->
<!--    <node pkg="landshark_spoofing" type="gps_spoofer" name="gps_spoofer" output="screen" />-->
<!--    <node pkg="landshark_spoofing" type="wheel_encoder_spoofer" name="wheel_encoder_spoofer" output="screen" />-->
<!--    <node pkg="landshark_spoofing" type="odometry_spoofer" name="odometry_spoofer" output="screen" />-->
<!--    <node pkg="landshark_spoofing" type="imu_spoofer" name="imu_spoofer" output="screen" />-->
<!--    -->
<!--    <param name="/sim/gps_topic" value="/gps_true" />-->
<!--    <param name="/sim/wheel_encoder_topic" value="/wheel_encoder_true" />-->
<!--    <param name="/sim/odometry_topic" value="/odom_true" />-->
<!--    <param name="/sim/imu_topic" value="/imu_true" />-->
<!--  </group>-->
 


</launch>
